HPXMLtoOpenStudio/resources/battery.rb

```diff
@@ -79,6 +88,7 @@ def self.apply(runner, model, pv_systems, battery, schedules_file, unit_multipli
     elcs.setNumberofStringsinParallel(number_of_strings_in_parallel)
     elcs.setInitialFractionalStateofCharge(0.0)
     elcs.setBatteryMass(battery_mass)
+    elcs.setDCtoDCChargingEfficiency(battery.round_trip_efficiency) # Note: This is currently unused in E+, so we use an EMS program below instead
```

Comment:
It was confusing to have the IDF show a round trip efficiency that doesn't match the HPXML inputs, so I added this in even though it doesn't affect the E+ results.